presidio-analyzer/presidio_analyzer/predefined_recognizers/in_pan_recognizer.py

```diff
@@ -26,17 +26,17 @@ class InPanRecognizer(PatternRecognizer):
     PATTERNS = [
         Pattern(
             "PAN (High)",
-            r"\b([A-Za-z]{3}[AaBbCcFfGgHhJjLlPpTt]{1}[A-Za-z]{1}[0-9]{4}[A-Za-z]{1})\b",
+            r"\b([A-Z]{3}[ABCFGHJLPT]{1}[A-Z]{1}(?!0000)[0-9]{4}[A-Z]{1})\b",
             0.85,
```

Comment:
Please reduce to 0.5. This is a strong regex, but could still produce false positives